llvm/include/llvm/IR/IntrinsicsAArch64.td

```diff
@@ -3107,6 +3107,26 @@ let TargetPrefix = "aarch64" in {
     }
   }
 
+  class SME_OuterProduct_TMOP_Intrinsic
+     : DefaultAttrsIntrinsic<[],
+         [llvm_i32_ty,
+          llvm_anyvector_ty,
+          LLVMMatchType<0>,
+          LLVMMatchType<0>,
+          llvm_nxv16i8_ty,
+          llvm_i32_ty],
+         [ImmArg<ArgIndex<0>>, ImmArg<ArgIndex<5>>,
+          IntrInaccessibleMemOnly]>;
+
+  def int_aarch64_sme_za16_ftmopa : SME_OuterProduct_TMOP_Intrinsic;
```

Comment:
nit: sorry for suggesting in the first place the wrong way, but just because I double check what is the name of the other sme with za instructions, and it looks like it should be
s/int_aarch64_sme_za16_ftmopa /int_aarch64_sme_ftmopa_za16/
and also for the other ones

Reply:
Sure, done 😀